Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift

```diff
@@ -211,14 +211,22 @@ class ProductDetailViewController: ButtonBarPagerTabStripViewController, DataMan
 
         createFormRow(with: &rows, item: product.categoriesTags?.map({ (categoryTag: String) -> NSAttributedString in
             if let category = dataManager.category(forTag: categoryTag) {
-                if let name = Tag.choose(inTags: Array(category.names)) {
+                if let name = Tag.choose(inTags: Array(category.names), forLanguageCode: Bundle.main.currentLocalization) {
```

Comment:
@teolemon 
I realized that Categories weren't being translated either, so I added `currentLocalization` from the Bundle. This works better than `preferredLocalization` because `currentLocalization` gets updated when the user changes their language settings.

Reply:
Instead of changing in every `Tag.choose` call, it would be better to replace the call to `preferredLocalizations.first` in the Tag.choose implementation, so that we have the same implementation and logic every where